### PR TITLE
Basic Rap Note Support

### LINKED ISSUE #479: Add Rap Notes (basic support)
Vocaluxe doesn't support Rap Notes at all, which USDX does since it's last release in 2017. Even worse, if a song contains a rap note, Vocaluxe ignores the whole song.
From my understanding, rap notes are skipping the note detection, helping to concentrate on the "Flow", which is great. Freestyle notes are not a real option, since there are songs with a lot of rap parts, and you want to have some nice highscores in these too...

I'll share two links with you, in case you want to take a look at it.
https://area91.garycxjk.com/usdx/rap
This is a guy, who implemented rap notes in USDX in the beginning of 2017.
https://github.com/UltraStar-Deluxe/USDX/pull/210
This is "his" pull, which was then merged into the build.
Maybe this helps you, understanding what is happening there.

I created a small video to show, what I mean. It's on my GDrive, here is the link:
https://drive.google.com/file/d/1ZpOmNFvigUOAsS0oykChQN__zuy3Ds2K/view?usp=sharing

### PR DESCRIPTION
**Pull request by ghost**
- Rap notes are displayed with the already existing texture for rap notes
- Rap notes are scored based on timing, ignoring the pitch

With this change, rap notes are now handled like in other similar programs. Of course there is room for improvement, but until someone has the time and skills to implement more advanced techniques (e.g. Rhythm detection as discussed here #479) this solution improves the current status of rap notes being freestyle notes.

### CODE REVIEW

File: VocaluxeLib/Menu/CLyric.cs [hunk 1]
```diff
@@ -118,7 +118,22 @@ public void SetLine(CSongLine line)
         private void _SetText(CSongNote note)
         {
             _Text.Text = note.Text;
-            _Text.Font.Style = (note.Type == ENoteType.Freestyle) ? EStyle.BoldItalic : EStyle.Bold;
+            if (note.Type == ENoteType.Freestyle)
```
Comment (flokuep) [inline VocaluxeLib/Menu/CLyric.cs]: I'd suggest to use a switch/case statement here or even combine the if-blocks into one as most conditions lead into the same code.

**Review by flokuep — CHANGES_REQUESTED** [review]
Thanks for your work on this topic! I'm just wondering what you want to achieve, as your code is just adjusting visual stuff and fixing a bug that rap notes aren't properly recognised. 
In your description, you've said that scoring of rap notes is also possible now. How is that done? Is it already working, but broken due to the rap note recognition bug?

**Comment by ghost** [comment]
> In your description, you've said that scoring of rap notes is also possible now. How is that done? Is it already working, but broken due to the rap note recognition bug?

Yes, the scoring already works without any necessary changes.

**Review by flokuep — APPROVED** [review]

**Comment by marwin89** [comment]
thanks for bug fixing! 👍🏻 👍🏻

**Pull request merged**